daemon/daemon_main.go

```diff
@@ -286,21 +287,36 @@ func checkMinRequirements() {
 	if err := os.MkdirAll(globalsDir, defaults.StateDirRights); err != nil {
 		log.WithError(err).WithField(logfields.Path, globalsDir).Fatal("Could not create runtime directory")
 	}
+
+	// Old Cilium BPF feature probes.
+	// TODO(mrostecki): Remove them completely and move missing checks to
```

Comment:
In a subsequent PR. We have some checks that are not implemented in bpftool, so I need to first send patches.

By the way, since we are already talking about that. Is it fine if I send the missing probes to the bpf list and add `Signed-off-by` of all authors who modified probe files, and the info that it comes from Cilium?

Reply:
I would probably only add `Signed-off-by` if I had asked the relevant authors if that was okay (which you could likely do via Slack). Otherwise there is `Authored-by` / `Co-authored-by` and/or just a note in the commit message pointing to the origin, `bpf/` directory license (which is GPLv2, hence compatible), and attributing the code to the relevant Cilium authors. It would also be good etiquette to CC the relevant people on the mail.